Sources/MapboxNavigation/PreviewViewController.swift

```diff
@@ -227,6 +227,20 @@ public class PreviewViewController: UIViewController, BannerPresentation {
                          completion: completion)
     }
     
+    // :nodoc:
+    public func dismissAllExceptFirst(at position: BannerPosition,
```

Comment:
Sometimes it's useful to pop all banners expect first one in the stack. Better naming suggestions are welcomed.